### PR TITLE
feat: Initial reports

### LINKED ISSUE #1: Initial Reports
Tracking issue for initial set of reports

### PR DESCRIPTION
**Pull request by graza-io**

### CODE REVIEW

**Comment by graza-io** [comment]
Changed target to be release branch instead of `main`

File: README.md [hunk 1]
```diff
@@ -1 +1,75 @@
...
+
+### Credentials
+
+This mod uses the credentials configured in the [Steampipe GitLab plugin](https://hub.steampipe.io/plugins/theapsgroup/gitlab).
+
+### Configuration
+
+No extra configuration is required.
+
+### FAQ
```
Comment (misraved) [inline README.md]: Is this section needed?

File: dashboards/group/docs/group_report_2fa.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+This report answers the following questions:
+
+- Which of my groups do/don't require 2fa?
```
Comment (misraved) [inline dashboards/group/docs/group_report_2fa.md]:
```suggestion
- Which of my groups do/don't require 2FA?
```

File: dashboards/group/docs/group_report_expiring_member.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+This report answers the following questions:
+
+- Do my groups have any members with expiring access?
+- When do members with expiring access lose it?
```
Comment (misraved) [inline dashboards/group/docs/group_report_expiring_member.md]:
```suggestion
- When does the access expire for the members?
```

File: dashboards/group/group_report_2fa.sp [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    where
+      not require_two_factor_authentication;
+  EOQ
+}
+
+query "group_2fa_table" {
+  sql = <<-EOQ
+    select
+      full_path as "Group",
+      require_two_factor_authentication as "2fa Required",
```
Comment (misraved) [inline dashboards/group/group_report_2fa.sp]:
```suggestion
      require_two_factor_authentication as "2FA Required",
```

File: dashboards/group/group_report_2fa.sp [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+      two_factor_grace_period as "Grace Period (in hours)",
```
Comment (misraved) [inline dashboards/group/group_report_2fa.sp]:
```suggestion
      two_factor_grace_period as "2FA Grace Period (in hours)",
```

File: dashboards/project/project_report_access_request.sp [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+      gitlab_my_project p
+    join
+      gitlab_project_access_request r
+    on p.id = r.project_id
+    where
+      r.state = 'active';
+  EOQ
+}
+
+query "project_access_request_table" {
```
Comment (misraved) [inline dashboards/project/project_report_access_request.sp:55]: Should we also sort the report based on `Age in Days`?

**Review by misraved — COMMENTED** [review]
@graza-io left minor review comments. Thanks!!

File: README.md (hunk 1 above)
Comment (graza-io) [reply README.md]: @misraved is this comment in reference to the `Configuration` or `FAQ` section?

**Review by misraved — APPROVED** [review]

**Pull request merged**